src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -146,9 +154,24 @@ export const thorchainApi: SwapperApi = {
         const publicClient = viemClientByChainId[chainId as EvmChainId]
         assert(publicClient !== undefined, `no public client found for chainId '${chainId}'`)
 
+        const expectedAmountOut = BigInt(longtailData?.longtailToL1ExpectedAmountOut ?? 0)
+        // Paranoia assertion - expectedAmountOut should never be 0 as it would likely lead to a loss of funds.
+        assert(
+          expectedAmountOut !== undefined && expectedAmountOut !== 0n,
```

Comment:
bigints in js are signed, so layer my paranoia with yours 🤝
```suggestion
          expectedAmountOut !== undefined && expectedAmountOut > 0n,
```

Reply:
🤝 https://github.com/shapeshift/web/pull/6052/commits/3c5c907403bda0f1fe7547a3eda5942c08de5ee3